charts/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -19,8 +19,6 @@ spec:
         image: "{{ .Values.schemaLoading.image.repository }}:{{ .Values.schemaLoading.image.version }}"
         imagePullPolicy: "{{ .Values.schemaLoading.image.pullPolicy }}"
         args:
-        - "-P"
```

Comment:
I see that `ContactPort` is only for cassandra.

Reply:
Yes, https://github.com/scalar-labs/scalardb/tree/master/schema-loader#run

Reply:
Is this change related to the JDBC support?

Reply:
Sorry for the inconvenience.

I have created a separate PR for changes outside of JDBC support